src/rpc-request.js

```diff
@@ -1,6 +1,15 @@
-import { Buffer } from 'buffer/';
 import { getBaseURL } from './utils';
 
+// [https://gist.github.com/999166] by [https://github.com/nignag]
+function b64(a, b = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=', c, d, e) {
+  for (
+    d = e = '';
+    a[d | 0] || (b = '=', d % 1);
+    e += b[63 & c >> 8 - d % 1 * 8]
+  ) c = c << 8 | a.charCodeAt(d -= -.75);
+  return e;
+}
+
```

Comment:
How about the following:
```js
function b64(str) {
    if (typeof btoa === 'undefined') {
        // Node doesn't have btoa, but does have Buffer
        return Buffer.from(str).toString('base64');
    }
    return btoa(str);
}
```
?

Reply:
umh, could maybe work. not sure if some build script would parse some javascript and build some AST tree and see "Ah, Buffer is being used, best we include a browserified version of buffer" and include it anyway.

Reply:
That seems pretty dubious to me, are there certain tools that are known to do this? Either way, we should not be including a copy of the npm buffer in our distributions.

Reply:
webpack prior to v5 automatically polyfilled node modules
https://github.com/webpack/changelog-v5/blob/master/README.md#automatic-nodejs-polyfills-removed

Think browserify dose it too.

but this will also depend upon how any other package uses this sdk - they maybe bundle in the sdk with there toolchain and maybe they polyfill node modules automatically.

Reply:
but if we would do something like what you suggest then i think that type checking should be done once.

```js
const b64 = typeof btoa === 'undefined' 
  ? str => Buffer.from(str).toString('base64') 
  : btoa
```